app/src/main/java/com/example/plantsapp/presentation/ui/notification/TaskNotificationManager.kt

```diff
@@ -136,6 +135,26 @@ class TaskNotificationManager @Inject constructor(
         )
     }
 
+    private fun Context.getPendingIntentForCompleteAction(
+        task: Task,
+        notificationId: Int,
+        requestCode: Int,
+        plantName: Plant.Name
+    ): PendingIntent {
+        return when (task) {
+            is Task.TakingPhotoTask -> clickPendingIntent
+            else -> {
+                NotificationBroadcastReceiver.createCompletePendingIntent(
+                    context = this,
+                    task = task,
+                    notificationId = notificationId,
+                    requestCode = requestCode,
+                    plantName = plantName
+                )
+            }
+        }
+    }
+
```

Comment:
1. Add //TODO - open camera on creation of task
2. rename it to something like getPendingIntentForTask